jnigen/java/src/test/resources/com/example/Example.java

```diff
@@ -5,28 +5,28 @@
 package com.example;
 
 public class Example {
-  static final boolean staticFinalField = true;
+  public static final boolean staticFinalField = true;
 
-  Example(int instanceField) {
+  public Example(int instanceField) {
     this.instanceField = instanceField;
   }
 
-  static String staticField = "hello";
+  public static String staticField = "hello";
 
-  static String getStaticField() {
+  public static String getStaticField() {
     return staticField;
   }
 
-  int instanceField;
+  public int instanceField;
 
-  int getInstanceField() {
+  public int getInstanceField() {
     return instanceField;
   }
 
   public static class Aux extends Example {
-    static int nothing = 0;
+    public static int nothing = 0;
 
-    static Example getAnExample() {
+    public static Example getAnExample() {
       return new Example();
     }
```

Comment:
Do we need to add a `private`, `protected`, and no-modifier method/field as well?

Also, what can and can't we call through JNI? Only `public`?

Reply:
It looks to be same as reflection, in that you can call non public methods. But we wouldn't want to expose private APIs either way.

Reply:
Well ... maybe users do want to do that at some point.

Shouldn't we just report all in the summarizer? (E.g. the summarizer should not be opinionated, it should just summarize.)

And then we filter on public in jnigen. And then if some user comes around that wants the private stuff, we add a flag.

Reply:
> Shouldn't we just report all in the summarizer?

Sounds like technically correct idea.

What should I do now? Implement it in current PR or add to issues?

Reply:
I have pushed a commit.